contracts/registry/src/name/normalized.rs

```diff
@@ -0,0 +1,169 @@
+use soroban_sdk::{Env, String};
+
+use super::to_str::AsStr;
+use crate::Error;
+
+pub(crate) struct Normalized {
+    len: usize,
+    internal: [u8; Self::MAX_NAME_LENGTH],
+}
+
+impl Normalized {
+    pub const MAX_NAME_LENGTH: usize = 64;
+
+    pub fn canonicalize(s: &String) -> Result<String, Error> {
+        let normalaized_str = Normalized::new(s)?;
```

Comment:
```suggestion
        let normalized_str = Normalized::new(s)?;
```
You have a typo in the variable name